R/bs4_book.R

```diff
@@ -48,7 +48,8 @@ bs4_book <- function(
                      ...,
                      lib_dir = "libs",
                      pandoc_args = NULL,
-                     extra_dependencies = NULL
+                     extra_dependencies = NULL,
+                     split_bib = split_bib
```

Comment:
This should be FALSE by default right ? 
```suggestion
                     split_bib = FALSE
```

Also when adding an argument, it should be added to the documentation here: 
https://github.com/rstudio/bookdown/pull/1185/files#diff-ff358cd3f877c575281978ccf49a2c8869cdcef59cdba125698dc3e8966a570aL41